content/lessons/chapter-10/opening-a-channel-4.tsx

```diff
@@ -12,48 +13,54 @@ export const metadata = {
 
 export default function OpeningAChannel4({ lang }) {
   const t = useTranslations(lang)
+  const [hydrated, setHydrated] = useState(false)
+  useEffect(() => {
+    setHydrated(true)
+  }, [])
 
   return (
-    <TransactionChallenge
-      initialStack={{
-        output_0: { 0: ['0', 'SIG(LASZLO)', 'SIG(YOU)'] },
-        output_1: { 0: [] },
-      }}
-      answerScript={{
-        output_0: [
-          'OP_PUSH',
-          'PUBKEY(LASZLO)',
-          'PUBKEY(YOU)',
-          'OP_CHECKMULTISIG',
-        ],
-        output_1: [],
-      }}
-      progressKey={metadata.key}
-      currentTransactionTab="multi-sig"
-      laszloWillNotSign
-      laszloHidden
-    >
-      <LessonInfo>
-        <Text className="text-lg font-bold md:text-xl">
-          {t('chapter_ten.opening_a_channel_four.heading_one')}
-        </Text>
-        <Text className="mt-4 text-lg md:text-xl">
-          {t('chapter_ten.opening_a_channel_four.paragraph_one')}
-        </Text>
-        <ul className="ml-4 list-disc  font-nunito text-lg">
-          <li>{t('chapter_ten.opening_a_channel_four.multisig_one')}</li>
-          <li>{t('chapter_ten.opening_a_channel_four.multisig_two')}</li>
-        </ul>
-        <Text className="mt-4 text-lg font-bold md:text-xl">
-          {t('chapter_ten.opening_a_channel_four.heading_two')}
-        </Text>
-        <ul className="ml-4 list-disc  font-nunito text-lg">
-          <li>
-            {t('chapter_ten.opening_a_channel_four.hint_one_a')}
-            {t('chapter_ten.opening_a_channel_four.hint_one_b')}
-          </li>
-        </ul>
-      </LessonInfo>
-    </TransactionChallenge>
+    hydrated && (
+      <TransactionChallenge
+        initialStack={{
+          output_0: { 0: ['0', 'SIG(LASZLO)', 'SIG(YOU)'] },
+          output_1: { 0: [] },
+        }}
+        answerScript={{
```

Comment:
How does this answer script work if it is missing the `OP_2` elements required for `OP_CHECKMULTISIG`?

Reply:
It just runs a `.includes()` method on the user's script array so the `OP_CHECKMULTISIG` already requires those `OP_2` commands

Reply:
oh, nice!